### ISSUE TITLE
project_name is not respected when doing successive runs

### ISSUE THREAD

**Issue by tpoisot** [Circuitscape/Omniscape.jl#146]
If the project name key is set to `x`, successive calls to `run_omniscape` will write files in `x`, `x_1`, `x_2`, ...

I get the idea of not losing outputs, but it's a little annoying as it's not documented anywhere and there doesn't seem to be an easy way to get the path of where the files are.

**Comment by vlandau** [comment]
Good point about not being able to get output. What do you propose? I could prompt for user input to confirm overwriting the files, or alternatively, I could log the location of the outputs more clearly and print them at the end of the run? There are already so damn many settings that I don't want to add another about overwriting existing outputs, but I suppose I could, just default it to false, and in those cases, print log warnings about existing outputs and the location of new outputs.

**Comment by vlandau** [comment]
@tpoisot I added a warning at the very beginning of a run that will let the user know if the directory already exists and print the location where outputs will be written. There is probably a more elegant solution, so I can leave this open if you'd like.
